README.md

```diff
@@ -41,7 +41,7 @@ The `useImage` hook allows for incorperating `react-image`'s logic in any compon
 Example usage:
 
 ```js
-import useImage from 'react-image'
```

Comment:
wont you need to update the rollup bundler somehow the export this file?

Reply:
Ive updated the readme

Reply:
i meant if you are using `react-image/useImage`, the file will need to be produced somewhere to be shipped to npm

currently im not seeing where that is happening